src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue18946.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue18946 : _IssuesUITest
+	{
+		public override string Issue => "Shell Toolbar items not displayed";
+
+		public Issue18946(TestDevice device) : base(device)
+		{ }
+
+		[Test]
+		public void ToolbarItemsShouldBeVisible()
+		{
+			var label = App.WaitForElement("label");
```

Comment:
The test is failing.

Reply:
I think it should work now :) I changed it to verify the screenshot because Appium couldn't find the Android toolbar item even though it works

![Screenshot_1718131226](https://github.com/dotnet/maui/assets/42434498/ecb85a8e-3d00-4d0c-a7f9-5ebd504c15df)